### DISCUSSION TITLE
Slack / IRC / other community help for beginners?

### DISCUSSION THREAD

**Discussion by vegabook** [yggdrasil-network/yggdrasil-go#996] [Q&A] [answered]
Hi, can't seem to find a slack or IRC or other community help forum. Does such a thing exist? 

I have successfully installed yggdrasil and am "talking" to peers (I think) because I can see yggdrasil sites from the [web directory](http://[21e:a51c:885b:7db0:166e:927:98cd:d186]/) but I have no idea if MY machine is visible (from behind my home networking NAT) nor how to setup my own home page (I'm proficient in web servers etc but not much with IPV6 nor yggdrasil's basic capabilities). 

Thanks.

**Comment by NikitaPuzyryov** [comment, marked as answer]
Yeah. A few options are listed in the [readme](https://github.com/yggdrasil-network/yggdrasil-go#community)
